test.js

```diff
@@ -18,3 +20,34 @@ assert(!ip.range('1.0.1.*').contains('1.0.0.10'));
 assert(ip.range('1.2.3.4-10').contains('1.2.3.4', '1.2.3.10'));
 assert(ip.range('1.2.3.5', '1.2.3.4').contains('1.2.3.4', '1.2.3.5'));
 
+// Handy tool for generating test data: http://www.ipaddressguide.com/cidr#range
+// If the range finishes with wildcards, we should have
+// one cidr notation range with a netmask.
+assert(ip.range('1.0.1.*').inetAddresses.length === 1);
```

Comment:
feel free to use `assert.equal` and `assert.notEqual` here for better failure messages ;)